recipes/gegca/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gegca" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gegca-{{ version }}.tar.gz
+  sha256: 6b8f9e3d384c3b5b30d3601f5501f500883dd3cc8138c1febbcb1d949d9c0f3d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9.0,<4.0.0
+    - pandas
+  run_constrained:
+    - matplotlib-base *
+    - notebook *
+
+test:
+  imports:
+    - gegca
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/znicholls/gegca
```

Comment:
This repo seems to contain just boilerplate. Where's the actual source code? 🤔

Reply:
See reply here https://github.com/conda-forge/staged-recipes/pull/27015#issuecomment-2236160739

Reply:
See https://github.com/conda-forge/staged-recipes/pull/27015#issuecomment-2236163323